docs/assets/cache_lifecycles.puml

```diff
@@ -43,15 +47,14 @@ Session -> "IK Cache" : get IK from cache
 Session -> API : complete operation 2
 |||
 API -> Session : close session
-Session -> "IK Cache" : securely close cache
-destroy "IK Cache"
+Session -> "Session Cache" : cache session
+deactivate "IK Cache"
 API -> Caller : API call User 1 result
 |||
 |||
 Caller -> API : API call User 2
 API -> SessionFactory : create new session
-SessionFactory -> Session : create new instance
-Session -> "IK Cache" : create new cache
+"Session Cache" -> Session : get cached session
```

Comment:
can we specify:
` if (cachedSession)
           get cached session
   else
            create new instance`